### PR TITLE
[Amusement Park Improvements]: Add test to match story

### PR DESCRIPTION
**Pull request by safwansamsudeen**
As [discussed](https://forum.exercism.org/t/add-test-to-amusement-park-improvements/5948).

### CODE REVIEW

**Review by kotp — COMMENTED** [review]
I wonder if this is the preferred order for the tests.

File: exercises/concept/amusement-park-improvements/attendee_test.rb [hunk 1]
```diff
@@ -35,6 +35,12 @@ def test_fits_ride_but_no_pass
     refute Attendee.new(100).allowed_to_ride?(100)
   end
 
+  def test_does_not_fit_ride_and_pass
+    attendee = Attendee.new(100)
+    attendee.issue_pass!(1)
+    refute attendee.allowed_to_ride?(120)
+  end
```
[the remark this answers is not in the document]
Comment (safwansamsudeen) [reply exercises/concept/amusement-park-improvements/attendee_test.rb:42]: I thought it starts off with one negative test case, then (my) another negative test case, then the happy path.

[the remark this answers is not in the document]
Comment (safwansamsudeen) [reply exercises/concept/amusement-park-improvements/attendee_test.rb:42]: Sure!

**Comment by safwansamsudeen** [comment]
Is this what you mean?

**Review by kotp — APPROVED** [review]
Looks good to me!

**Comment by kotp** [comment]
Thank you @safwansamsudeen !

**Pull request merged**